Failure to detect shumlib on Cray

Compiling with the Cray compiler fails to detect shumlib:

CMake Error at /home/users/david.davies/cylc-run/mo-bundle-414/share/mobbs/util/ex_cray/share/cmake-4.0/Modules/FindPackageHandleStandardArgs.cmake:227 (message):
  Could NOT find SHUM (missing: SHUM_INCLUDE_DIR)
Call Stack (most recent call first):
  /home/users/david.davies/cylc-run/mo-bundle-414/share/mobbs/util/ex_cray/share/cmake-4.0/Modules/FindPackageHandleStandardArgs.cmake:591 (_FPHSA_FAILURE_MESSAGE)
  opsinputs/cmake/FindSHUM.cmake:79 (find_package_handle_standard_args)
  opsinputs/CMakeLists.txt:72 (find_package)

--- a/cmake/FindSHUM.cmake
+++ b/cmake/FindSHUM.cmake
@@ -34,17 +34,10 @@ To guide the search process, set the SHUM_ROOT environment variable or CMake var
 
 #]=======================================================================]
 
-if(CMAKE_Fortran_COMPILER_ID STREQUAL "Cray")
-  find_path(SHUM_INCLUDE_DIR
-    NAMES F_SHUM_FIELDSFILE_CLASS_VERSION_MOD.mod
-    PATH_SUFFIXES ${SHUM_PATH}/include
-  )
-else()
-  find_path(SHUM_INCLUDE_DIR
-    NAMES f_shum_fieldsfile_class_version_mod.mod
-    PATH_SUFFIXES ${SHUM_PATH}/include
-  )
-endif()
+find_path(SHUM_INCLUDE_DIR
+  NAMES F_SHUM_FIELDSFILE_CLASS_VERSION_MOD.mod f_shum_fieldsfile_class_version_mod.mod
+  PATH_SUFFIXES ${SHUM_PATH}/include
+)
 
 find_library(SHUM_BYTESWAP_LIBRARY NAMES shum_byteswap)
 find_library(SHUM_STRING_CONV_LIBRARY NAMES shum_string_conv)

--- a/cmake/FindSHUM.cmake
+++ b/cmake/FindSHUM.cmake
@@ -34,10 +34,17 @@ To guide the search process, set the SHUM_ROOT environment variable or CMake var
 
 #]=======================================================================]
 
-find_path(SHUM_INCLUDE_DIR
-  NAMES f_shum_fieldsfile_class_version_mod.mod
-  PATH_SUFFIXES ${SHUM_PATH}/include
-)
+if(CMAKE_Fortran_COMPILER_ID STREQUAL "Cray")
+  find_path(SHUM_INCLUDE_DIR
+    NAMES F_SHUM_FIELDSFILE_CLASS_VERSION_MOD.mod
+    PATH_SUFFIXES ${SHUM_PATH}/include
+  )
+else()
+  find_path(SHUM_INCLUDE_DIR
+    NAMES f_shum_fieldsfile_class_version_mod.mod
+    PATH_SUFFIXES ${SHUM_PATH}/include
+  )
+endif()
 
 find_library(SHUM_BYTESWAP_LIBRARY NAMES shum_byteswap)
 find_library(SHUM_STRING_CONV_LIBRARY NAMES shum_string_conv)

